curriculum/challenges/english/25-front-end-development/quiz-html-accessibility/66ed9026f45ce3ece4053ebb.md

```diff
@@ -223,67 +258,87 @@ The `aria-labelledby` attribute allows you to give an element an accessible name
 
 #### --text--
 
-What is the function of the `aria-selected` attribute?
+Which of the following ARIA states is used to indicate that an element is selected?
 
 #### --distractors--
 
-To change the font size of an element.
+`aria-haspopup`
 
 ---
 
-To change the background color of an element.
+`aria-label`
 
 ---
 
-To shrink an element.
+`aria-expanded`
 
 #### --answer--
 
-To indicate if an element is selected.
+`aria-selected`
 
 ### --question--
 
 #### --text--
 
-What is the function of `aria-expanded`?
+Which of the following is NOT a common ARIA state?
 
 #### --distractors--
 
-To change the font size of an element.
+`aria-haspopup`
```

Comment:
```suggestion
`aria-hidden`
```
Can we use `aria-hidden` here instead. `aria-haspopup` is technically a property, not a state.